examples/40-access-entries.yaml

```diff
@@ -21,6 +21,7 @@ accessConfig:
       kubernetesGroups: # optional Kubernetes groups
         - group1 # groups can used to give permissions via RBAC
         - group2
+      kubernetesUsername: username # optional Kubernetes username
```

Comment:
these are just examples, but the config schema does show/document this option https://eksctl.io/usage/schema/#accessConfig-accessEntries-kubernetesUsername